clouds/tests/aws/test_aws_utils.py

```diff
@@ -0,0 +1,123 @@
+from typing import Dict, List
+from unittest.mock import patch
```

Comment:
please use `mocker` and not `unittest.mock`

Reply:
done